Index: src/renderer.cpp
===================================================================
--- src/renderer.cpp	(revision 100)
+++ src/renderer.cpp	(working copy)
@@ -125,7 +125,7 @@
 		else
 			setGlobalOption("Variants", "Primary Readings");
 			
-		LocaleMgr::systemLocaleMgr.setDefaultLocaleName(options.locale());
+		LocaleMgr::getSystemLocaleMgr()->setDefaultLocaleName(options.locale());
 	}
 	
 	/** Return an HTML hyperlinked list of all modules,
@@ -447,7 +447,7 @@
 			return;
 		
 		// FIXME - why do I need this call to setLocale()?
-		vk->setLocale(LocaleMgr::systemLocaleMgr.getDefaultLocaleName());
+		vk->setLocale(LocaleMgr::getSystemLocaleMgr()->getDefaultLocaleName());
 		
 		modtextdir = textDirection(module);
 			
@@ -820,7 +820,7 @@
 		
 		if (!vk)
 			return output;
-		vk->setLocale(LocaleMgr::systemLocaleMgr.getDefaultLocaleName());
+		vk->setLocale(LocaleMgr::getSystemLocaleMgr()->getDefaultLocaleName());
 			
 		module->setSkipConsecutiveLinks(true);
 		vk->AutoNormalize(1);
@@ -1000,7 +1000,7 @@
 	
 	QStringList Renderer::availableLocales()
 	{
-		list<SWBuf> locales = LocaleMgr::systemLocaleMgr.getAvailableLocales();
+		list<SWBuf> locales = LocaleMgr::getSystemLocaleMgr()->getAvailableLocales();
 		list<SWBuf>::const_iterator it;
 		list<SWBuf>::const_iterator it_end = locales.end();
 		QStringList output;
